repos/core-setup.common.props

```diff
@@ -21,6 +21,7 @@
     <BuildArguments>$(BuildArguments) /p:RestoreAllBuildRids=false</BuildArguments>
     <BuildArguments>$(BuildArguments) /p:OutputRid=$(OverrideTargetRid)</BuildArguments>
     <BuildArguments>$(BuildArguments) /p:DotNetOutputBlobFeedDir=$(SourceBuiltBlobFeedDir)</BuildArguments>
+    <BuildArguments>$(BuildArguments) /p:SkipSetLastReleasedVersionForSourceBuild=true</BuildArguments>
```

Comment:
I suggest linking https://github.com/dotnet/runtime/issues/641 in a comment for intent.

Reply:
Fixed.